GrandCentralBoard/Widgets/Harvest/View/AreaBarChartViewModel.swift

```diff
@@ -33,9 +33,21 @@ struct AreaBarChartViewModel {
     let componentCharts: [AreaBarChartComponentViewModel]
 
     let horizontalAxisStops: Int
-    let horizontalAxisLabelText: String
+    let horizontalAxisLabelText: String = "people billing:"
 
     let centerText: String?
+    let historicalHeaderText: String
+
+    init(mainChart: AreaBarChartComponentViewModel, componentCharts: [AreaBarChartComponentViewModel], horizontalAxisStops: Int,
+         centerText: String?) {
+        self.mainChart = mainChart
+        self.componentCharts = componentCharts
+        self.horizontalAxisStops = horizontalAxisStops
+        self.centerText = centerText
+
+        let headerTextFormat = componentCharts.count == 1 ? "PREVIOUS %d DAY" : "PREVIOUS %d DAYS"
```

Comment:
Two things:
- `Previous day` is imo more correct grammatically
- `"Previous %d Days". uppercaseString` looks better to me